package.json

```diff
@@ -87,6 +87,10 @@
     "ad-block": "^3.0.2",
     "aphrodite": "1.1.0",
     "async": "^2.0.1",
+    "bat-balance": "^0.9.6",
+    "bat-client": "^0.9.0",
+    "bat-publisher": "^0.9.0",
+    "bignumber.js": "^4.0.4",
```

Comment:
[minor] there's a new version of bignumber.js out now https://github.com/MikeMcl/bignumber.js/